src/vs/workbench/parts/terminal/electron-browser/windowsShellHelper.ts

```diff
@@ -56,7 +68,7 @@ export class WindowsShellHelper {
 				} else {
 					const childProcessLines = stdout.split('\n').slice(1).filter(str => !/^\s*$/.test(str));
 					const childProcessDetails = childProcessLines.map(str => {
-						const s = str.split('  ');
+						const s = str.split(/\s{2,}/);
```

Comment:
It was because in cases where wmic doesn't know the process name there are more than 2 spaces before the process id.